feat(sbb-chip-group): initial implementation (#3382)
sbb-design-systems/lyne-components: src/elements/chip-group/chip-group/readme.md

The `sbb-chip-group` component is a container for one or multiple `sbb-chip`.
Generally, it is used in combination with an `sbb-form-field` to allow the input of multiple string values.

The `value` property reflects the list of slotted chips. Adding or removing an `sbb-chip` updates the value property, and vice versa.

```html
<sbb-form-field>
  <label>Field label</label>
  <sbb-chip-group name="field-name">
    <sbb-chip value="Value 1"></sbb-chip>
    ...
    <input />
  </sbb-chip-group>
</sbb-form-field>
```

## Slots

Use the unnamed slot to provide the `sbb-chip` and the `input` field.

## States

The `sbb-chip-group` has a `disabled` and a `readonly` state and reacts to the respective `input` properties.

> The `disabled`/`readonly` property is proxied to the slotted `sbb-chip`.

Reviewer: ```suggestion
The `disabled`/`readonly` properties are proxied to the slotted `sbb-chip`s.
```

</sbb-form-field>
```

### Custom 'separator-keys'

> By default, the `sbb-chip-group` create a new chip on `Enter` key press.

Reviewer 2: ```suggestion
By default, the `sbb-chip-group` creates a new chip on `Enter` key press.
```

> Consumers can customize the array of [keys](https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values#special_values) that will trigger the chip creation by using the `SeparatorKeys` property.

Reviewer 2: I think we should use the real property name if talking about property
```suggestion
Consumers can customize the array of [keys](https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values#special_values) that will trigger the chip creation by using the `separatorKeys` property.
```
